### PR TITLE
Fix READMEs for all the packages: soft-wrap and line-break

### PR DESCRIPTION
**Pull request by kariharju**
### This PR
- Fixes _README.md_ of every package with soft-wraps (no line width limit) and rendered line breaks (`  `) according to Markdown's author himself.
- Adds release notes for every affected package (later with [commit](https://github.com/robocorp/robocorp/commit/2a6a3249d6782a1a5a70099005881d8708447fb7)) -> next release will make the new README visible in PyPI.
### Notes
- @kariharju: The linefeeds by writer in _README.md_ files are creating a havoc on the docs -site side. When possible, use IDE's "Word Wrap" (soft-wrap) when writing Markdown to avoid manually introducing hard-wraps.
- @cmin764: At the same time, I took the chance to fix the bugs and refactor these READMEs, so they'd follow this [example](https://www.notion.so/robocorptechnologies/Markdown-fff71182e16d441b8599a40266ec9cc8?pvs=4#59d20b51b7894b7587a615b544601222) with soft-wrap and line breaks introduced by double-spaces (`  `).

### Up Next
- Make releases of every affected package, including **robocorp** (_meta_) and Action Server:
  - _actions_
  - _browser_
  - _log_ & _log_pytest_
  - _storage_
  - _tasks_
  - _vault_
  - _windows_
  - _workitems_

### CODE REVIEW

File: browser/README.md
Comment (cmin764) [inline browser/README.md]:
```suggestion
> - _action-package.yaml_ for automation Action Packages
```

File: windows/README.md
Comment (cmin764) [inline windows/README.md]:
```suggestion
> - _action-package.yaml_ for automation Action Packages
```

**Review by cmin764 — CHANGES_REQUESTED** [review]
LGTM given the content, but detected two other typos. (commented below)

Now given the line length, according to `markdownlint` [docs](https://github.com/markdownlint/markdownlint/blob/main/docs/RULES.md#md013---line-length), a default line length of 80 chars is expected, but ofc, we can increase that.

What do you think @fabioz , shall we add a linter for the MDs we write/generate? Something that has a prettier preferably?

**Comment by cmin764** [comment]
This may look blocked now by this issue: https://github.com/robocorp/robocorp/issues/257

But to unblock it, maybe we should manually edit these files to match what we would achieve with the solution above, then merge it as it is without staying blocked until the solution is ready.

⚠️ What I still don't understand, is why our docs site rendering part isn't compatible with the Markdown suggested [format & rules](https://github.com/markdownlint/markdownlint/blob/main/docs/RULES.md#md013---line-length) or why this started to become a problem today when we wrote line breaks in _.md_ files the entire time? Isn't this a problem that should be treated on the [**documentation**](https://github.com/robocorp/documentation) side, as long as we enforce a commonly accepted code-driven standard here in **robocorp**?

**Comment by kariharju** [comment]
The length rule is not a markdown rule, just a markdown linter rule.
The docs rendering is only breaking if there are extra spaces, and the only way to get that is to manually add linefeeds.

Markdown is not code, it is content to applying old code styling rules on it really does not make sense. Markdown will in all cases (except looking at raw data in a text editor) be shown in whatever width layout so applying a some limit in the source file really does not help. 

Need to get these fixes in and let's look at the #257 separately.

**Comment by cmin764** [comment]
Will refactor this PR by applying this Notion [proposal](https://www.notion.so/robocorptechnologies/Markdown-fff71182e16d441b8599a40266ec9cc8?pvs=4#855177f5d45145adafa56c36ed8a29a0).

**Pull request merged**
